subsys/zigbee/Kconfig

```diff
@@ -200,6 +200,24 @@ config ZIGBEE_UART_TX_BUF_LEN
 
 endif #ZIGBEE_HAVE_SERIAL
 
+menuconfig ZIGBEE_CLI_SHELL
+	bool "Enable Zigbee CLI Shell"
+	select SHELL
+	imply SHELL_BACKEND_SERIAL
```

Comment:
No particular reason, the idea was to have at least one backend enabled when Zigbee CLI is enabled and serial is probably the most default of all. It can be removed so it is up to user to choose which backend to enable when enabling the Zigbee CLI for an application.

Reply:
So enabling `SHELL` does not link any sort of `SHELL_BACKEND`?